actix-http/src/extensions.rs

```diff
@@ -66,6 +66,11 @@ impl Extensions {
     pub fn extend(&mut self, other: Extensions) {
         self.map.extend(other.map);
     }
+
+    /// Returns `true` if no extension is registered
+    pub fn is_empty(&self) -> bool {
```

Comment:
lets limit public API

```suggestion
    pub(crate) fn is_empty(&self) -> bool {
```